__tests__/features/end-to-end.steps.ts

```diff
@@ -557,6 +559,19 @@ defineFeature(loadFeature("./end-to-end.feature"), test => {
         .sendKeys("Health notes");
       await browser!.submit();
 
+      // Disability page
+      await expect(browser!.getCurrentUrl()).resolves.toContain(
+        "/disability?process_type=thc"
+      );
+      await browser!.findElement({ id: "disability-yes" }).click();
+      await browser!.findElement({ id: "who-disability-tenant-1" }).click();
+      await browser!.findElement({ id: "pip-or-dla-yes" }).click();
+      await browser!.findElement({ id: "who-pip-tenant-1" }).click();
+      await browser!
+        .wait(until.elementLocated({ name: "disability-notes" }), 500)
+        .sendKeys("Disability notes");
```

Comment:
Could just be like `await browser!
        .waitForEnabledElement({ name: "door-mats-notes" })
        .sendKeys("Door mats notes");`
I'm assuming (touch wood) that `waitForEnabledElement` does the wait inside it.

Reply:
It does :).